bin/module2rpm

```diff
@@ -98,15 +98,15 @@ multi MAIN(:$module!, Bool :$debug, Bool :$dryrun, :$dir) {
     my $module-metadata = %all-metadata{$module};
     die "Did not find metadata for module '$module'" unless $module-metadata;
 
-    my $spec = Module2Rpm::Spec.new(metadata => $module-metadata);
-    my $package = Module2Rpm::Package.new(spec => $spec, path => $destination-path);
+    my $metadata = Module2Rpm::Metadata.new(metadata => $module-metadata);
+    my $package = Module2Rpm::Package.new(metadata => $metadata, path => $destination-path);
 
     $log.info("Download module source...");
     $dir.IO.mkdir if $dir and not $dir.IO.e;
```

Comment:
This was changed in the previous PR. However, I think it would be nice to create a temporary directory with tempdir(), if $dir was not passed.  $dir is not mandatory after all.